testing/trino-product-tests/src/main/java/io/trino/tests/product/hudi/TestHudiSparkCompatibility.java

```diff
@@ -320,6 +320,44 @@ public void testTimelineTableRedirect()
         }
     }
 
+    @Test(groups = {HUDI, PROFILE_SPECIFIC_TESTS})
+    public void testReadCopyOnWriteTableWithReplaceCommits()
+    {
+        String tableName = "test_hudi_cow_replace_commits_select_" + randomNameSuffix();
+
+        onHudi().executeQuery(format(
+                """
+                        CREATE TABLE default.%s (
+                          id bigint,
+                          name string,
+                          price int,
+                          ts bigint)
+                        USING hudi
+                        TBLPROPERTIES (
+                          type = '%s',
+                          primaryKey = 'id',
+                          preCombineField = 'ts',
+                          hoodie.clustering.inline = 'true',
+                          hoodie.clustering.inline.max.commits = '1')
+                        LOCATION 's3://%s/%s'""",
+                tableName,
+                COW_TABLE_TYPE,
+                bucketName,
+                tableName));
+
+        onHudi().executeQuery("INSERT INTO default." + tableName + " VALUES (1, 'a1', 20, 1000), (2, 'a2', 40, 2000)");
```

Comment:
Move to inside of `try`. Otherwise, the table won't be dropped if this statement failed for some reason.

Reply:
Ok, done.